compiler-plugin/src/main/java/io/ballerina/stdlib/persist/compiler/DiagnosticsCodes.java

```diff
@@ -73,6 +73,7 @@ public enum DiagnosticsCodes {
             "references for relation", ERROR),
     PERSIST_123("PERSIST_123", "associated entity does not contain any primary keys: the key " +
             "should have a valid value", ERROR),
+    PERSIST_124("PERSIST_124", "invalid initialization: the entity should be a closed record", ERROR),
```

Comment:
Give entity name.
```suggestion
    PERSIST_124("PERSIST_124", "The entity ''{0}'' should be a closed record.", ERROR),
```

Reply:
Fixed in [f857258](https://github.com/ballerina-platform/module-ballerina-persist/pull/70/commits/f857258a161209a9f3228bde6dc3d96214d768e4)